test/contrib/http/request_test.rb

```diff
@@ -0,0 +1,112 @@
+require 'time'
+require 'contrib/elasticsearch/test_helper'
+require 'ddtrace'
+require 'helper'
+require 'json'
+
+class HTTPRequestTest < Minitest::Test
+  ELASTICSEARCH_HOST = '127.0.0.1'.freeze
+  ELASTICSEARCH_PORT = 49200
+  ELASTICSEARCH_SERVER = ('http://' +
+                          HTTPIntegrationTest::ELASTICSEARCH_HOST + ':' +
+                          HTTPIntegrationTest::ELASTICSEARCH_PORT.to_s).freeze
+
+  def setup
+    @tracer = get_test_tracer
+
+    # wait until it's really running, docker-compose can be slow
+    wait_http_server 'http://' + ELASTICSEARCH_HOST + ':' + ELASTICSEARCH_PORT.to_s, 60
+
+    @client = Net::HTTP.new(ELASTICSEARCH_HOST, ELASTICSEARCH_PORT)
+    pin = Datadog::Pin.get_from(@client)
+    pin.tracer = @tracer
+  end
+
+  def test_get_request
+    response = @client.get('_cluster/health')
+    assert_equal('200', response.code, 'bad response status')
+    content = JSON.parse(response.body)
+    assert_kind_of(Hash, content, 'bad content')
+    spans = @tracer.writer.spans()
+    assert_equal(1, spans.length)
```

Comment:
@ufoot this means that if the ElasticSearch and the HTTP integrations are both enabled, we will have 2 traces for the same logical action?